Tasks/PublishSymbols/PublishSymbols.ps1

```diff
@@ -130,6 +136,10 @@ try {
             [IO.File]::AppendAllLines($tmpFileName, [string[]]@($fullFilePath))
         }
 
+        [string] $encodedRequestName = [System.Web.HttpUtility]::UrlEncode($RequestName)
+        [string] $requestUrl = "$SymbolServiceUri/DefaultCollection/_apis/Symbol/requests?requestName=$encodedRequestName"
```

Comment:
This task will be used only in hosted (non-onprem) scenario, and we dont have (or accept) the account name or collection information.
Removed the literal hardcoded name.